src/annotator/test/notebook-test.js

```diff
@@ -0,0 +1,90 @@
+import Notebook from '../notebook';
+
+describe('Notebook', () => {
+  // `Notebook` instances created by current test
+  let notebooks;
+
+  const createNotebook = (config = {}) => {
+    config = { notebookAppUrl: '/base/annotator/test/empty.html', ...config };
+    const element = document.createElement('div');
+    const notebook = new Notebook(element, config);
+
+    notebooks.push(notebook);
+
+    return notebook;
+  };
+
+  beforeEach(() => {
+    notebooks = [];
+  });
+
+  afterEach(() => {
+    notebooks.forEach(n => n.destroy());
+  });
+
+  describe('notebook container frame', () => {
+    it('starts hidden', () => {
+      const notebook = createNotebook();
+      assert.equal(notebook.container.style.display, 'none');
+    });
+
+    it('displays when opened', () => {
```

Comment:
For tests of more than 3/4 lines I find it can help readability to separate the Arrange/Act/Assert phases in a test by a new line, at least for cases where the test statements neatly fall into that pattern. I admit the existing frontend code is not super consistent/rigorous about this.

Reply:
I've added some groupings throughout this test module now.